js/gated-content/src/components/personal-training/modal/UserName.vue

```diff
@@ -26,25 +26,46 @@
 </template>
 <script>
 import Modal from '@/components/modal/Modal.vue';
+import client from '@/client';
 import { mapGetters, mapActions } from 'vuex';
 
 export default {
   components: { Modal },
+  data() {
+    return {
+      error: null,
+      name: '',
+    };
+  },
   computed: {
     ...mapGetters([
-      'localName',
       'isShowUserNameModal',
     ]),
   },
+  created() {
+    client
+      .get('personal-training/get-user-name')
```

Comment:
Why do we need this code here, since we are doing the same at Store part?

Reply:
@anpolimus in store we have the `post` request